students/gene/project/node_db.py

```diff
@@ -0,0 +1,131 @@
+
+import sqlite3
+from sqlite3 import Error
+
+
+class Node:
+
+    def __init__(self, node_id, site_id='', management_ip='', console_id='', vendor_id='', platform_id='', os_id=''):
+        self.node_id = node_id
+        self.site_id = site_id
+        self.management_ip = management_ip
+        self.console_id = console_id
+        self.vendor_id = vendor_id
+        self.platform_id = platform_id
+        self.os_id = os_id
+
+
+    @property
+    def description(self):
+        return ("""Node: {}
+Site: {}
+Management IP: {}
+Console: {}
+Vendor: {}
+Platform: {}
+SW Version: {}""".format(self.node_id, self.site_id, self.management_ip,
+                      self.console_id, self.vendor_id, self.platform_id,
+                      self.os_id))
+
+
+class NodeStorage:
+    """
+    To test in database in memory use ':memory:'
+    To use database use 'test.db'
+
+    """
+
+    def __init__(self, database='test.db'): 
+        self.db = database
+        self.conn = sqlite3.connect(self.db)

```

Comment:
maybe this should call self.create_connection() instead? Better not to have the same code in more than one place.